Lares/Assets/Mechanics/Inventory/InventoryItem.cs

```diff
@@ -4,6 +4,17 @@
 
 namespace Lares.Inventory
 {
+    public class ObjectID : PropertyAttribute { }
+
+    [Serializable]
+    public struct InventoryData
+    {
+        [ObjectID] public string ItemID;
+        public string itemName;
+        public string itemDescription;
+        public UnityEngine.UI.Image itemImage;
```

Comment:
```suggestion
        [ObjectID] public string ItemId;
        public string ItemName;
        public string ItemDescription;
        public UnityEngine.UI.Image ItemImage;
```

Any references to the old names will need refactoring.